streams/fr_groupem6.m3u

```diff
@@ -11,3 +11,11 @@ http://190.2.155.162:8080/m6music/mpegts
 https://shls-tiji-tv-prod-dub.shahid.net/out/v1/98f46736bd8c4404b67e4b7a38cc8976/index.m3u8
 #EXTINF:-1 tvg-id="TiJi.fr",Tiji (1080p)
 https://shls-tiji-tv-prod-dub.shahid.net/out/v1/ee05878a88474f408ff04495d44cc249/index.m3u8
+#EXTINF:-1 tvg-id="M6.fr",M6
+https://raw.githubusercontent.com/Sphinxroot/HSL/main/M6.m3u8
+#EXTINF:-1 tvg-id="Gulli.fr",Gulli
+https://origin2-6play.live.6cloud.fr/pool_rnpls9by/gulli/sixplaysd/hls_short_q2hyb21h_gulli/index.m3u8
```

Comment:
Gulli in HD also is https://origin2-6play.live.6cloud.fr/pool_rnpls9by/gulli/sixplayhde/hls_short_fp10_gulli/NA/vZNA6hM/index.m3u8 + https://origin2-6play.live.6cloud.fr/pool_rnpls9by/gulli/sixplayhde/hls_short_fp10_gulli/NA/aVWvbjp/index.m3u8